qa/rpc-tests/getrawtransaction_insight.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014-2015 The Bitcoin Core developers
```

Comment:
I think if there are other insight related rpc tests it may be convenient to have the name be insight_getrawtransaction, but I don't think this is a big deal, and can be changed later.